Troubleshoot/TroubleshootError_AcsEngine.ps1

```diff
@@ -0,0 +1,439 @@
+#
+# ClassifyError.ps1
+#
+<# 
+    .DESCRIPTION 
+		Classifies the error type that a user is facing with their Acs-Engine Kubernetes cluster
+ 
+    .PARAMETER SubscriptionId
+        Subscription Id that the Acs-Engine Kubernetes cluster is in
+
+    .PARAMETER ResourceGroupName
+        Resource Group name where the Acs-Engine Kubernetes cluster is in
+    
+#>
+
+param(
+    [Parameter(mandatory = $true)]
+    [string]$SubscriptionId,
+    [Parameter(mandatory = $true)]
+    [string]$ResourceGroupName	
+)
+
+$ErrorActionPreference = "Stop";
+Start-Transcript -path .\TroubleshootDump.txt -Force
+$OptOutLink = "https://github.com/helm/charts/tree/master/incubator/azuremonitor-containers"
```

Comment:
Optional: https://github.com/helm/charts/tree/master/incubator/azuremonitor-containers#uninstalling-the-chart